### PR TITLE
TA-410:  Fix tag master on pact publish

### PR DESCRIPTION
**Pull request by ruban72**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by Vamshi-Muniganti — APPROVED** [review]
looks good

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Comment by ruban72** [comment]
Close

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -51,7 +51,7 @@ withPipeline(type, product, component) {
 
    enablePactAs([
      AppPipelineDsl.PactRoles.CONSUMER
-   ])
+])
```
Comment (paul-graham) [inline Jenkinsfile_CNP:54]: What's happened here, eh? Looks like someone deleted something they shouldn't have.

**Review by udaydenduluri33 — APPROVED** [review]
LGTMe

File: config/custom-environment-variables.json [hunk 2]
```diff
@@ -91,5 +91,12 @@
     "sessionTimeouts": {
       "__name": "SESSION_TIMEOUTS",
       "__format": "json"
+    },
+    "pact": {
+      "brokerUrl": "PACT_BROKER_URL",
+      "consumerVersion": "PACT_CONSUMER_VERSION",
+      "branchName": "PACT_BRANCH_NAME",
+      "brokerUsername": "PACT_BROKER_USERNAME",
+      "brokerPassword": "PACT_BROKER_PASSWORD"
```
Comment (paul-graham) [inline config/custom-environment-variables.json:100]: This password is weak and easily guessed. 😞

**Review by paul-graham — APPROVED** [review]
No need for Jenkinsfile_CNP to have changed - and certainly not in the way it has - but otherwise all appears to be above board.

**Pull request merged**